### PR TITLE
Existing documentation converted to .md

### PR DESCRIPTION
**Pull request by zbrydon**
# Description

Converted existing DreamBig documentation to .md

## Type of change

Documentation 

# How Has This Been Tested?

NA

## Testing Checklist:

- [ ] Tested in latest Chrome
- [ ] Tested in latest Safari
- [ ] Tested in latest Firefox

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have requested a review from Tan Le on the Pull Request

### CODE REVIEW

**Review by tancnle — CHANGES_REQUESTED** [review]
Thank you for your hard work @zbrydon 👍🏼 Could you kindly run the formatter to fix up the styles (see [instruction](https://github.com/thoth-tech/documentation/blob/main/CONTRIBUTING.md))?

For future reference, please break big PR to smaller ones to allow quicker review and reduce cognitive load of the reviewers.

File: docs/DreamBig/Project Overview (Epic and Sprint Plan)/DreamBig Prototype Epic.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+## Acceptance criteria 
+
+(Informal)
+
+- Validate documentation with managing directors
+- Documentation produced is accurate
+
+
+
+
```
Comment (tancnle) [inline docs/DreamBig/Project Overview (Epic and Sprint Plan)/DreamBig Prototype Epic.md:77]: **issue:** These empty lines caused the build to fail and do not allow merge. Can you remove them?

Comment (zbrydon) [reply docs/DreamBig/Project Overview (Epic and Sprint Plan)/DreamBig Prototype Epic.md]: Ah was wondering why the check was failing, just removed them.

**Review by NehanjaliMakineni — COMMENTED** [review]
Sorry for my late review
@zbrydon hoping all the issues mentioned by @tancnle are addressed?

File: docs/DreamBig/Sprint 2 documentation/Wireframes/WireFrame.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Group 1 WireFrame
+
+Guanyu Chen
+
+## Log in page
+
+![Login Page](./../../assests/images/login_g1.jpg)
```
Comment (tancnle) [inline docs/DreamBig/Sprint 2 documentation/Wireframes/WireFrame.md:7]: **suggestion:** Link must not start with "./"

```suggestion
![Login Page](../../assests/images/login_g1.jpg)
```

File: docs/DreamBig/Sprint 1 documentation/Individual Research/UI_Design&Frontend_Research_HarryLiu.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+HTML5 is the simplest way of front-end development, which has the advantage of being easier to learn
+and requires less data to be stored when developing, which makes the development cost go down. In
+addition, HTML5 can be easily adapted to Web, ios and Android without spending too much time on
+interface adaptation issues. But its disadvantage is the user experience line tea, because the HTML5
+application is poor in operation, the performance ability is general, so in terms of user experience
+and performance HTML5 has a great shortage, and lack of application smooth feeling.
+
+CSS is a style sheet language, it is used to express the HTML file style language, the role is used
+for static modification of web pages, such as modify the color, adjust the layout, modify the font
```
Comment (tancnle) [inline docs/DreamBig/Sprint 1 documentation/Individual Research/UI_Design&Frontend_Research_HarryLiu.md:15]: **suggestion:** We should stick to Australian English.

```suggestion
for static modification of web pages, such as modify the colour, adjust the layout, modify the font
```

File: docs/DreamBig/Sprint 2 documentation/Wireframes/Dream Big Wireframes (Gautam).md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Dream Big Wireframes
+
+![Login](../../assests/images/gautam_login.jpg)
+
+This is the login page for the dream big application. Every Deakin student have their Deakin login
+credentials so we can use those credentials to login to our dream big web application.
```
Comment (tancnle) [inline docs/DreamBig/Sprint 2 documentation/Wireframes/Dream Big Wireframes (Gautam).md:6]:
```suggestion
credentials so we can use those credentials to log in to our dream big web application.
```

File: docs/DreamBig/Sprint 1 documentation/Dream Big functionality reqs brainstorm - Thoth-Tech.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+| Reflections                    | On activities/outcomes <br> Show progress regarding employability <br> Show avatar changes + spiderweb presentation as reflections are provided/activities completed <br> Can include details related to careers/jobs/employability ideas stored in profile <br> Links to LinkedIn and other university services/resources for guidance                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+
+![White Board Image](./../assests/images/MicrosoftTeams-image.png)
```
Comment (tancnle) [inline docs/DreamBig/Sprint 1 documentation/Dream Big functionality reqs brainstorm - Thoth-Tech.md:11]:
```suggestion
![White Board Image](../assests/images/MicrosoftTeams-image.png)
```

**Review by tancnle — COMMENTED** [review]
Thanks, @zbrydon. Thank you for bearing with me. We have implemented additional linting as part of https://github.com/thoth-tech/documentation/pull/60. As a result, could you please manually address some outstanding issues, please 🙏🏼 ?

**Review by tancnle — APPROVED** [review]
Thanks for the prompt action @zbrydon. Look good to me. Let's merge 🚀

**Comment by zbrydon** [comment]
No worries @tancnle, just went through and made those changes.

**Pull request merged**